src/Symfony/Component/Translation/Translator.php

```diff
@@ -290,11 +290,11 @@ protected function getLoaders()
      *
      * @return array[array] indexed by catalog
      *
-     * @deprecated since version 2.8, to be removed in 3.0. Use TranslatorBagInterface::getCatalogue() method instead.
+     * @deprecated since version 2.8, to be removed in 3.0. Rely on TranslatorBagInterface::getCatalogue() method instead.
```

Comment:
it was suggested by stof see https://github.com/symfony/symfony/pull/14546#discussion_r29592981